packages/workflow/src/interfaces.ts

```diff
@@ -456,3 +456,18 @@ export type DefaultSignalHandler = (signalName: string, ...args: unknown[]) => v
  * A validation function capable of accepting the arguments for a given UpdateDefinition.
  */
 export type UpdateValidator<Args extends any[]> = (...args: Args) => void;
+
+/**
+ * A description of a signal or query handler.
+ */
+export type SignalOrQueryHandlerOptions = { description?: string };
+
+/**
+ * A validator and description of an update handler.
+ */
+export type UpdateHandlerOptions<Args extends any[]> = { validator?: UpdateValidator<Args>; description?: string };
+
+/**
+ * Options for a generic handler.
+ */
+export type AnyHandlerOptions<Args extends any[]> = UpdateHandlerOptions<Args>;
```

Comment:
Wouldn't it make more sense for `AnyHandlerOptions` to be `{ description?: string }`, with `SignalOrQueryHandlerOptions` being an alias to that (for now, possibly forever), and `UpdateHandlerOptions` being an extension of `AnyHandlerOptions` to include `validator?` ?

Reply:
I'm not even sure it is really pertinent to make a base common interface. Yes, turns out that Query, Signal and Update all accept a `description`, but do we ever care about that common denominator?

Reply:
@antlai-temporal Let's make this instead:

```
export type QueryHandlerOptions = { description?: string };

export type SignalHandlerOptions = { description?: string };

export type UpdateHandlerOptions<Args extends any[]> = { validator?: UpdateValidator<Args>; description?: string };
```

... and no `AnyHandlerOptions`. Just use a union of those types where appropriate.

Reply:
Yeah, I'm fine with that too.